app/components/menu/menu.css

```diff
@@ -0,0 +1,23 @@
+.bb-menu {
```

Comment:
kinda weird to use the "bb-" prefix here and not anywhere else. i get this is probably to avoid collisions, but we should probably have a more complete strategy

could we just call this thing dropdown or dropdown-menu instead?

Reply:
How about just "menu-list" since it's really just a wrapper around `ul role="menu"`?